minitest/repositores/company_repository.py

```
@@ -0,0 +1,24 @@
+from .base_repository import BaseRepository
+from ..tmp_database import tmp_company_db
```

Ruff (0.8.2)

`..tmp_database.tmp_company_db` imported but unused

Remove unused import: `..tmp_database.tmp_company_db`

(F401)